hirola/front/static/front/js/dashboard.js

```diff
@@ -14,18 +14,18 @@ function editName(hiddenInput, visibleInput){
 var detailsLink = document.getElementById('details-link');
 var detailsRow = document.getElementById('details');
 
-detailsLink.addEventListener('click',() => {
-    if(!detailsRow.style.display || detailsRow.style.display == 'none'){
-        detailsRow.style.display = 'block';
-    }else{
-        detailsRow.style.display = 'none';
-    }
-});
+// detailsLink.addEventListener('click',() => {
```

Comment:
Delete the commented out methods on the js file. As they are not being used there is no need to keep them.